website/content/docs/blueprints/core.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Core Components"
```

Comment:
Similar to `core` and `addOn`, we should also include `kubernetes` section as well - that should give users idea that they can add Kubernetes section to have bctl bootstrap Kubernetes

Reply:
I agree. I can see this section getting large in order to cover all of the different blueprint sections. I only added these first as placeholders since I could see doing all of our current blueprint sections as a PR by itself